ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomCommandingDemoController.swift

```diff
@@ -54,20 +54,35 @@ class BottomCommandingDemoController: UIViewController {
         CommandingItem(title: "Boolean Item " + String($0), image: homeImage, action: commandAction, isToggleable: true)
     }
 
-    private lazy var expandedListSections: [CommandingSection] = [
+    private lazy var fewExpandedListSections: [CommandingSection] = [
         CommandingSection(title: "Section 1", items:
         Array(1...2).map {
             CommandingItem(title: "Item " + String($0), image: homeImage, action: commandAction)
-        } + booleanCommands),
-        CommandingSection(title: "Section 2", items: Array(1...7).map {
+        } + booleanCommands)
+    ]
+
+    private lazy var manyExpandedListSections: [CommandingSection] = fewExpandedListSections + [
+        CommandingSection(title: "Section 2", items:
```

Comment:
the toggle didn't seem additive.  shouldn't it start with section 1?

Reply:
It reuses the first section of hero commands from the fewExpandedListSections array.